tldraw/tldraw
# improvement(stickies): add slight translucency to show shape behind it

Add slight translucency to sticky notes to show shapes behind them.

- 0.98 felt too subtle
- 0.95 too obvious
- 0.96 a bit too much still
- 0.97 felt just right to me, indicating that this was intentionally translucent without being obvious about it

https://github.com/tldraw/tldraw/assets/469604/45869141-a588-4a1a-9864-270d3602d3ef

### Change type


### Test plan

1. Create a shape (e.g. a rectangle).
2. Place a sticky note over the shape.
3. Verify that the shape behind the sticky note is slightly visible through it.


### Release notes

- Stickies: add slight translucency.
